packages/jest-cli/src/__tests__/watch-filename-pattern-mode-test.js

```diff
@@ -145,6 +145,8 @@ describe('Watch mode flows', () => {
   });
 
   it('can select a specific file name from the typeahead results', () => {
+    const toUnixPathPattern = pathPattern => pathPattern.replace(/\\\\/g, '/');
```

Comment:
This was the original problem the function is trying to fix

![970c1bf0-341a-11e7-92d2-09fc3f05c9de](https://cloud.githubusercontent.com/assets/7337433/26127703/fbd3e1fe-3a81-11e7-9032-0685296a729a.png)

Due to the backslashs needing escaping, the way I understand it is behind the hood "path\\to\\file" is actually "path\\\\to\\\\file", if that makes sense?